lib/node.js

```diff
@@ -499,9 +482,27 @@ declare class events$EventEmitter {
   setMaxListeners(n: number): void;
 }
 
+declare class EventEmitter extends events$EventEmitter {};
```

Comment:
this makes `EventEmitter` a valid global, which is not true is it? you have to import it from `events`?

would doing `declare class domain$Domain extends events$EventEmitter {` work?